packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -842,3 +842,274 @@ const Map<int, PhysicalKeyboardKey> kFuchsiaToPhysicalKey = <int, PhysicalKeyboa
   0x000c028b: PhysicalKeyboardKey.mailForward,
   0x000c028c: PhysicalKeyboardKey.mailSend,
 };
+
+/// Maps macOS-specific IDs to the matching [LogicalKeyboardKey].
+/// MacOS doesn't provide a scan code, but a virtual keycode to represent a physical key.
+/// Therefore, the LogicalKeyboardKey and PhysicalKeyboardKey maps are the same.
```

Comment:
Got it. Now I'm generating the Logical key from the unmodified character in raw_keyboard_macos.dart. I removed this map altogether since it's just duplicated from the physical keys map.